core/modules/commands/savewikifolder.js

```diff
@@ -158,11 +158,26 @@ WikiFolderMaker.prototype.saveCustomPlugin = function(pluginTiddler) {
 };
 
 WikiFolderMaker.prototype.saveTiddler = function(directory,tiddler) {
+	var title = tiddler.fields.title, fileInfo, pathFilters, extFilters;
+	if(this.wiki.tiddlerExists("$:/config/FileSystemPaths")){
+		pathFilters = this.wiki.getTiddlerText("$:/config/FileSystemPaths","").split("\n");
+	}
+	if(this.wiki.tiddlerExists("$:/config/FileSystemExtensions")){
+		extFilters = this.wiki.getTiddlerText("$:/config/FileSystemExtensions","").split("\n");
+	}
 	var fileInfo = $tw.utils.generateTiddlerFileInfo(tiddler,{
 		directory: path.resolve(this.wikiFolderPath,directory),
-		wiki: this.wiki
+		wiki: this.wiki,
+		pathFilters: pathFilters,
+		extFilters: extFilters,
+		originalpath: this.wiki.extractTiddlerDataItem("$:/config/OriginalTiddlerPaths",title, "")
 	});
-	$tw.utils.saveTiddlerToFileSync(tiddler,fileInfo);
+	try {
+		$tw.utils.saveTiddlerToFileSync(tiddler,fileInfo);
+	} catch (err) {
+		console.log("SaveWikiFolder: Error saving file '" + fileInfo.filepath + "', tiddler: '" + tiddler.fields.title);
+		console.error(err);
```

Comment:
We should, but we don't currently consistently send error messages to console.error. Perhaps that's something we should do in bulk as a separate PR?

Reply:
Agree. I think it would be more consistent to use $tw.logger.alert() here (& in similar cases), and that logger method should print errors to both node and browser console (along with the alert-tiddler logic). That should be a separate PR.